### PR TITLE
Updated the npm module for published ABI

### PR DESCRIPTION
**Pull request by julien51**
The npm module now has the ABI for both the Unlock.sol and PublicLock.sol contracts.

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [X] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by nfurfaro** [comment]
I'm just wondering what use-case you have in mind that requires the old Unlock abi? Is it just for consistency, and to remove the `unlock/versions` directory?

**Comment by hardlydifficult** [comment]
This depends on #1843 or similar

**Comment by julien51** [comment]
@nfurfaro Sure! Well I think we should decouple the development cycles for the smart contract and the web app.
I think it is brittle that we currently develop the web apps locally against the "current" work in the smart contract, when in practice the web app is eventually deployed against an "older" version of the smart contract. (see https://github.com/unlock-protocol/unlock/issues/1842)

So, I think we should package our bytecode and develop against these versions (as well as deploy these versions locally when developing the web apps).

I think we can/should get rid of `unlock/versions`.

**Comment by julien51** [comment]
I have updated the PR to reflect the change in the json file name so that it can be imported correctly.

File: smart-contracts/published-npm-modules/V0/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Unlock ABI v0
+
+This npm module includes the ABI for the smart contracts Unlock.sol and PublicLock.sol for v0.
+The smart contracts have been compiled as of commit d2a3cc65.
+
+This module is meant to be used by application which want to interract with Unlock's smart contracts as of v0.
```
Comment (nfurfaro) [inline smart-contracts/published-npm-modules/V0/README.md]: "applications"

Comment (julien51) [reply smart-contracts/published-npm-modules/V0/README.md]: Oups! Thanks for the sharp eye!

**Pull request merged**